src/components/DayPickerRangeController.jsx

```diff
@@ -217,6 +221,8 @@ export default class DayPickerRangeController extends React.PureComponent {
         chooseAvailableDate,
       },
       visibleDays,
+      disablePrev: false,
```

Comment:
can we prepopulate this given the initial props so that it's accurate on initial render?